cmd/moco-agent/cmd/ping.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+const (
+	credentialConfPathFlag = "credential-conf-path"
+)
+
+var pingCmd = &cobra.Command{
+	Use:   "ping",
+	Short: "Send ping to a MySQL instance",
+	Long:  `Send ping to a MySQL instance.`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		_, err := os.Stat("/var/lib/mysql/init-once-completed")
```

Comment:
https://github.com/cybozu-go/moco-agent/blob/bebff9ddd80b425da6a5df75575f53fb2ac27775/initialize/init_test.go#L19
Can we use this variable?